src/tdastro/astro_utils/cosmology.py

```diff
@@ -0,0 +1,64 @@
+import astropy.cosmology.units as cu
+from astropy import units as u
+
+from tdastro.base_models import FunctionNode
+
+
+def redshift_to_distance(redshift, cosmology, kind="comoving"):
+    """Compute a source's distance given its redshift and a
+    specified cosmology using astropy's redshift_distance().
+
+    Parameters
+    ----------
+    redshift : `float`
+        The redshift value.
+    cosmology : `astropy.cosmology`
+        The cosmology specification.
+    kind : `str`
+        The distance type for the Equivalency as defined by
+        astropy.cosmology.units.redshift_distance.
+
+    Returns
+    -------
+    distance : `float`
+        The distance (in pc)
+    """
+    z = redshift * cu.redshift
+    distance = z.to(u.pc, cu.redshift_distance(cosmology, kind=kind))
+    return distance.value
```

Comment:
I believe we need no other type of distance than luminosity distance. I propose to use `distance` parameter name everywhere, while specifying in the docs that it is "luminosity distance"

Reply:
I added the luminosity specifier to the doc strings.

Reply:
As discussed in person, I also changed the astropy function call and removed the `kind` option.